packages/react-composites/tests/browser/chat/app/FakeAdapterApp.tsx

```diff
@@ -27,17 +39,56 @@ export const FakeAdapterApp = (): JSX.Element => {
     verifyParamExists(params.fakeChatAdapterModel, 'fakeChatAdapterModel')
   ) as ChatAdapterModel;
 
+  // Optional params
+  const useFileSharing = Boolean(params.useFileSharing);
```

Comment:
avoid `use*` prefix unless this is a react hook
```suggestion
const fileSharingEnabled = Boolean(params.fileSharingEnabled);
```
or if this is doing something more specific e.g. attaching a message to the send box, name this to have that specificity:
```suggestion
const attachFileToInitialSendMessage = Boolean(params.attachFileToInitialSendMessage);
```